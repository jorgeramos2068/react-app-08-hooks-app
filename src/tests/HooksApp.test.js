import React from 'react';
import {shallow} from 'enzyme';
import HooksApp from '../HooksApp';

describe('Tests for <HooksApp />', () => {
  test('Should display <HooksApp /> correctly', () => {
    const wrapper = shallow(<HooksApp />);
    expect(wrapper).toMatchSnapshot();
  });
});
