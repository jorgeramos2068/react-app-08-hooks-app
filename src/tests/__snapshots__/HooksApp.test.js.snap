// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for <HooksApp /> Should display <HooksApp /> correctly 1`] = `
<div>
  <h1>
    HooksApp
  </h1>
</div>
`;
